words/compile.asm

```diff
@@ -44,6 +44,17 @@
 ; The inline forms are typically much simpler since they can use
 ; 65c02 jmp and bxx branch opcodes directly.
 
+;TODO does it make sense to have an actual word, like COMPILE-NT, ?
+compile_nt_comma:  ; ( nt -- )
```

Comment:
avoid the `int>name` by starting from nt and moving `name>int` inside.

(btw is `int` here short for `internal` or something?  I keep wanting to call these words `nt>xt` and `xt>nt` :)

Reply:
I suspect the int is for interpret.  ANS forth has a word NAME>INTERPRET to get the XT for the interpretation semantics for a word and NAME>COMPILE to get the XT for the compiling semantics for a word (on systems that have separate XTs for each scenario).